trimmer_decorate.rb

```diff
@@ -0,0 +1,7 @@
+require './decorate'
+
+class TrimmerDecorator < Decorate
+  def correct_name
+    decorate.correct_name.lenght <= 10 ? decorate.correct_name : decorate.correct_name[0..9]
```

Comment:
- [ ] There is no variable named `decorate`. You spelt `length` wrong also. Kindly fix that.

  ```
  @nameable.correct_name.length <= 10 ? @nameable.correct_name : @nameable.correct_name[0..9]
  ```